domains/certificates/Query.API/API/Query.API/Projections/CertificatesByOwnerProjection.cs

```diff
@@ -1,119 +1,186 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using API.Query.API.ApiModels.Responses;
+using System.Threading;
+using System.Threading.Tasks;
+using API.Query.API.Projections.Exceptions;
+using API.Query.API.Projections.Views;
+using Baseline;
 using CertificateEvents;
+using Marten;
+using Marten.Events;
 using Marten.Events.Projections;
-using Marten.Schema;
+using Marten.Linq.LastModified;
 
 namespace API.Query.API.Projections;
 
-public class CertificatesByOwnerProjection : MultiStreamAggregation<CertificatesByOwnerView, string>
+public class CertificatesByOwnerProjection : IProjection
 {
-    public CertificatesByOwnerProjection()
+    private static readonly List<CertificatesByOwnerView> views = new();
```

Comment:
By marking `views` as static, it means that it is bound to the class and not an instance of the class. I do not think that it should be static here.